xcode/Subconscious/Shared/Models/StoryEntry.swift

```diff
@@ -11,8 +11,7 @@ import Foundation
 struct StoryEntry:
     Hashable,
     Identifiable,
-    CustomStringConvertible,
-    Codable
```

Comment:
`Subtext` is not `Codable` and `EntryStub.excerpt` is `Subtext` now, so we have to drop this protocol.